.gitignore

```diff
@@ -1,2 +1,3 @@
 /node_modules
-/dist
\ No newline at end of file
+/dist
+/.env
```

Comment:
Although you have added this, you still need to delete your env file and commit the delete to remove it from the PR